test/test_signals.py

```diff
@@ -192,3 +260,111 @@ def test_signal_example(monkeypatch):
     # lambda expression returning None but only for this one test.
     monkeypatch.setattr(plt, "show", lambda: None, raising=True)
     demonstrate_signal()
+
```

Comment:
We should now extend the test suite by calling all methods with their allowed parameter sets.

Reply:
So, this is what is actually left now...

Reply:
Finally we got it! (https://app.codecov.io/gh/PTB-M4D/PyDynamic/blob/improve_and_repair_signal_class/src/PyDynamic/signals.py)